var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -78,6 +84,7 @@ class Curl(AutotoolsPackage):
             # 'schannel',
             "secure_transport",
             # 'wolfssl',
+            "sspi",
```

Comment:
Do you know if any of these are Windows-only? Linux-only? IMO worth using `conditional()` or `conflicts()` in those cases.